### ISSUE TITLE
Add Aries Framework Go Test Vectors to Spec

### ISSUE THREAD

**Issue by OR13** [w3c-ccg/webkms#8]
https://github.com/hyperledger/aries-framework-go/pull/2314

I believe there is possibly more implementation support than spec support at this point.

We should get test vectors committed here, to prevent implementations from drifting to far apart.

**Comment by troyronda** [comment]
We are adding a client implementation to AFG (as @OR13 referenced above).

We are also working on our WebKMS server implementation here: https://github.com/trustbloc/kms

**Comment by peacekeeper** [comment]
I'm looking at this spec, and while it sounds really useful, the spec and the two implementations mentioned here seem to be completely out of sync.

**Comment by OR13** [comment]
@peacekeeper yes, this has been my experience as well... I don't believe there is a "webkms" standard yet... or even a complete spec with more than 1 implementation.

**Comment by dlongley** [comment]
Yes, the spec needs to be brought up to date with current implementations. I believe there are at least three implementations in the wild. Digital Bazaar is currently updating its implementation with lessons learned around addressing root zcaps. We plan on updating the spec once that's finished as soon as we can find some time to do so.
